src/zenml/integrations/mlflow/experiment_trackers/mlflow_experiment_tracker.py

```diff
@@ -74,13 +79,17 @@ def my_step(context: StepContext, ...)
             `tracking_password` must be specified.
         tracking_insecure_tls: Skips verification of TLS connection to the
             MLflow tracking server if set to `True`.
+        databricks_host: The host of the Databricks workspace with the MLflow
+            managed server to connect to. This is only required if
+            `tracking_uri` value is set to `"databricks"`.
     """
 
     tracking_uri: Optional[str] = None
     tracking_username: Optional[str] = SecretField()
     tracking_password: Optional[str] = SecretField()
     tracking_token: Optional[str] = SecretField()
     tracking_insecure_tls: bool = False
+    databricks_host: Optional[str] = SecretField()
```

Comment:
The secret field right now warns users that they should specify the value as a secret, and in the future might even raise an exception if they don't. I'm not familiar with Databricks, but from a quick search the URL doesn't necessarily contain any sensitive information right? In that case I'd suggest making this just a normal `Optional[str]` field similar to the `tracking_uri` (it can still be specified as a secret, just isn't mandatory anymore).

Reply:
You're right, this one doesn't need to be a secret